server/src/internalClusterTest/java/org/opensearch/index/SegmentReplicationPressureIT.java

```diff
@@ -129,9 +129,8 @@ public void testWritesRejected() throws Exception {
         assertReplicaCheckpointUpdated(primaryShard);
         // index another doc showing there is no pressure enforced.
         indexDoc();
-        refresh(INDEX_NAME);
+        waitForReplication(true, INDEX_NAME);
```

Comment:
I don't think we need an additional wait here if we were already waiting for searchable dos on the next line?  This test was waiting for doc count and then validating the segments themselves.